go/test/endtoend/vtgate/setstatement/udv_test.go

```diff
@@ -46,7 +46,7 @@ func TestSetUDV(t *testing.T) {
 		expectedRows: "", rowsAffected: 0,
 	}, {
 		query:        "select @foo, @bar, @baz",
-		expectedRows: `[[VARCHAR("abc") INT64(42) DECIMAL(30.5)]]`, rowsAffected: 1,
+		expectedRows: `[[VARBINARY("abc") INT64(42) FLOAT64(30.5)]]`, rowsAffected: 1,
```

Comment:
I don't understand why this is changing.

Reply:
set statement e2e test are not added in config.json. This should have changes when I modified varchar to be stored as varbinary in bindvars. As this is not part of regular run, this did not fail.